lib/js/src/protocol/_SdlProtocolBase.js

```diff
@@ -268,6 +270,33 @@ class _SdlProtocolBase {
         return this._messageID++;
     }
 
+    /**
+     * Gets the Vehicle details received in StartService ACK protocol message
+     * @returns {VehicleType|null} - A RPC VehicleType struct received from the packet if exists null otherwise.
+     */
+    _getVehicleType(sdlPacket) {
```

Comment:
The documentation here is missing the @param tag for the `sdlPacket` parameter. There also needs to be a space before the function parentheses on line 277. Use the command `npm run lint` to confirm.

Reply:
Fixed